libtenzir/src/healthmetrics_collector.cpp

```diff
@@ -0,0 +1,101 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2023 The Tenzir Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <tenzir/data.hpp>
+#include <tenzir/detail/weak_run_delayed.hpp>
+#include <tenzir/healthmetrics_collector.hpp>
+#include <tenzir/import_stream.hpp>
+#include <tenzir/series_builder.hpp>
+
+#include <ranges>
+
+namespace tenzir {
+
+namespace {
+
+auto collect_metrics(
+  const healthmetrics_collector_state::collectors_map& collectors)
+  -> table_slice {
+  auto now = std::chrono::system_clock::now();
+  auto b = series_builder{
+    type{"tenzir.metrics.system", record_type{}, {{"internal", ""}}}};
```

Comment:
Can you use `metrics_name()` here in the schema name and document the available options just like [`tenzir.metrics.operator`](https://docs.tenzir.com/metrics#tenzirmetricsoperator)?

Reply:
You mean switch to one import stream per metric, as opposed to building the combined format?

Reply:
My understanding is that the import stream can handle heterogeneous events, is that incorrect?

Reply:
Yes, I think that's preferable, both for documentation and for usage in charts.

Reply:
Unresolving as this is still missing:

> [...] and document the available options just like [`tenzir.metrics.operator`](https://docs.tenzir.com/metrics#tenzirmetricsoperator)?